packages/ui/src/components/Avatar.tsx

```diff
@@ -0,0 +1,2 @@
+export { Avatar, AvatarGroup } from '@chakra-ui/react';
+export type { AvatarProps } from '@chakra-ui/react';
```

Comment:
👍 Great idea to re-export these components instead of importing them from Chakra in the TeamCard file.